### PR TITLE
Disable the IAPHelperTests

### PR DESCRIPTION
**Pull request by emilylaguna**
The test is still being flaky, so we'll disable it for now. 

## To test

🟢 CI

## Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [x] I have considered adding unit tests for my changes.
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

File: PocketCastsTests/UnitTests.xctestplan [hunk 1]
```diff
@@ -52,6 +52,10 @@
   },
   "testTargets" : [
     {
+      "skippedTests" : [
+        "IAPHelperTests",
```
Comment (bjtitus) [inline PocketCastsTests/UnitTests.xctestplan:56]: Isn't this first one enough to disable the whole file?

File: PocketCastsTests/Tests/IAP/IAPHelperTests.swift [hunk 2]
```diff
@@ -28,7 +28,7 @@ final class IAPHelperTests: XCTestCase {
         session.clearTransactions()
         SKPaymentQueue.default().remove(helper)
     }
-
+/*
```
Comment (bjtitus) [inline PocketCastsTests/Tests/IAP/IAPHelperTests.swift:31]: Does this need to be commented out when the whole file is disabled from the test plan?

**Review by bjtitus — APPROVED** [review]
Left a few comments but it fixes the currently broken test so I'll approve. ✅

File: PocketCastsTests/UnitTests.xctestplan (hunk 1 above)
Comment (emilylaguna) [reply PocketCastsTests/UnitTests.xctestplan:56]: I used the test plan to skip it in my first commit, but it still ran 🤷‍♂️ .

Comment (bjtitus) [reply PocketCastsTests/UnitTests.xctestplan:56]: Weird, it skipped for me locally. Better to be safe than sorry, I guess! 😅

**Pull request merged**